lib/services/presets.go

```diff
@@ -202,6 +196,50 @@ func NewPresetAuditorRole() types.Role {
 	return role
 }
 
+// NewPresetReviewerRole returns a new pre-defined role for reviewer. The
+// reviewer will be able to review all access requests.
+func NewPresetReviewerRole() types.Role {
+	enterprise := modules.GetModules().BuildType() == modules.BuildEnterprise
+	role := &types.RoleV6{
+		Kind:    types.KindRole,
+		Version: types.V6,
+		Metadata: types.Metadata{
+			Name:        teleport.PresetReviewerRoleName,
+			Namespace:   apidefaults.Namespace,
+			Description: "Review access requests",
+		},
+		Spec: types.RoleSpecV6{
+			Allow: types.RoleConditions{
+				ReviewRequests: defaultAllowAccessReviewConditions(enterprise)[teleport.PresetReviewerRoleName],
+			},
+		},
+	}
+	role.SetLogins(types.Allow, []string{"no-login-" + uuid.New().String()})
+	return role
+}
+
+// NewPresetRequesterRole returns a new pre-defined role for requester. The
+// requester will be able to request all resources.
+func NewPresetRequesterRole() types.Role {
+	enterprise := modules.GetModules().BuildType() == modules.BuildEnterprise
```

Comment:
Hmm now that I'm thinking about this, having "empty" requester and reviewer roles in OSS would be pretty confusing. I think we should do this:

- In OSS do not create these presets at all instead of creating empty roles as they're pretty useless without allow clauses.
- In the docs, only add "requester" and "reviewer" roles when Enterprise scope is selected.

Reply:
Okay, this has been done (save any docs syntax issues).